notebooks/Numba_052_release_demo.ipynb

```diff
@@ -0,0 +1,221 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@njit(debug=True) # Switch on debug to make python source available.\n",
+    "def foo(n):\n",
+    "    acc = 0.\n",
+    "    for i in range(n):\n",
+    "        acc += np.sqrt(i)\n",
+    "    if acc > 1000:\n",
+    "        raise ValueError(\"Error!\")\n",
+    "    else:\n",
+    "        return acc\n",
+    "\n",
+    "foo(10)\n",
+    "\n",
+    "# Take a look at the docstring for all the options, the ones used here are:\n",
+    "# strip_ir = remove LLVM IR apart from calls\n",
+    "# interleave = add Python source into the LLVM CFG!\n",
+    "foo.inspect_cfg(foo.signatures[0], strip_ir=True, interleave=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Newly supported NumPy functions/features\n",
+    "====================================\n",
+    "\n",
+    "This release contains some updates to Numba's NumPy support, mostly contributed by the Numba community (with thanks!):\n",
+    "* NumPy 1.19 ([@stuartarchibald](https://github.com/stuartarchibald)).\n",
+    "* ``np.asfarray`` ([@guilhermeleobas](https://github.com/guilhermeleobas)).\n",
+    "* \"subtyping\" in record arrays ([@luk-f-a](https://github.com/luk-f-a)).\n",
+    "* ``np.split`` and ``np.array_split`` ([@ivirshup](https://github.com/ivirshup)).\n",
+    "* ``operator.contains`` with ``ndarray`` ([@mugoh](https://github.com/mugoh)).\n",
+    "* ``np.asarray_chkfinite`` ([@rishabhvarshney14](https://github.com/rishabhvarshney14)).\n",
+    "* the ``ndarray`` allocators, ``empty``, ``ones`` and ``zeros``, accept a ``dtype`` specified as a string literal ([@stuartarchibald](https://github.com/stuartarchibald))."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "@njit\n",
+    "def demo_numpy():\n",
+    "    # np.asfarray\n",
+    "    farr = np.asfarray(np.zeros(4,), dtype=np.int8)\n",
+    "    \n",
+    "    # np.split/np.array_split\n",
+    "    split = np.split(np.arange(10), 5)\n",
+    "    arr_split = np.array_split(np.arange(10), 3)\n",
+    "    arr_contains = 4 in np.arange(10), 11 in np.arange(10)\n",
+    "\n",
+    "    # asarray_chkfinite\n",
+    "    caught = False\n",
+    "    try:\n",
+    "        np.asarray_chkfinite((0., np.inf, 1., np.nan,))\n",
+    "    except Exception: # inf and nan not accepted\n",
+    "        caught = True\n",
+    "\n",
+    "    # String literal dtypes\n",
+    "    ones, zeros, empty = (np.ones((5,), 'int8'), np.zeros((3,), 'complex128'),\n",
+    "                          np.empty((0,), 'float32'))\n",
+    "\n",
+    "    return farr, split, arr_split, arr_contains, caught, ones, zeros, empty\n",
+    "    \n",
+    "demo_numpy()"
```

Comment:
The output is a bit hard to read given there are so many. I'd prefer it goes like:

```python
print(f"farr: {farr}\nsplit: {split}...")
```